ansys_optical_automation/application/camera_nhtsa_test.py

```diff
@@ -0,0 +1,193 @@
+import os
+
+from ansys_optical_automation.speos_process.speos_sensors import Camera
+from ansys_optical_automation.speos_process.speos_simulations import Simulation
+
+
+def import_car(path, position_value):
+    param = GetActiveWindow().Groups[0]
```

Comment:
@BenoitHeraud please add numpy type docstring to explain your function

Reply:
@pluAtAnsys @StefanThoene which kind of information details do we need in such kind of description? a single quote? few key words?